src/components/workspace-utils.js

```diff
@@ -32,11 +32,12 @@ export const useWorkspaces = () => {
   const signal = useCancellation()
   const [loading, setLoading] = useState(false)
   const workspaces = useStore(workspacesStore)
+  const ajax = useAjax()
   const refresh = _.flow(
     withErrorReporting('Error loading workspace list'),
     Utils.withBusyState(setLoading)
   )(async () => {
-    const ws = await Ajax(signal).Workspaces.list([
+    const ws = await ajax(signal).Workspaces.list([
```

Comment:
it might feel arbitrary or hard for a future dev to understand why this and only this method is swapped over to the useAjax variant.  I would suggest a comment to call out why this one is singled out.

Reply:
I went with `useReplaceableAjax`. That name describes the intent. I think the word "experimental" tends to describe a temporary state, which is a prediction about the future, thus to be avoided.

Reply:
good name :)